### PR TITLE
Cut and download clip

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ ARG public_base="https://kabbalahmedia.info/"
 ARG feed_api_url="https://kabbalahmedia.info/feed_api/"
 ARG personal_api_url="https://kabbalahmedia.info/backend/my/"
 ARG chronicles_url="https://chronicles.kli.one/"
+ARG file_trimmer_api_url="https://trim.kab.sh/rest/trim"
```
Comment (davgur) [inline Dockerfile:6]: look like its not enough, can you help with this

File: src/helpers/Api.js [hunk 2]
```diff
@@ -250,6 +251,15 @@ class Api {
     const config = { url, method: 'GET' };
     return axios(config);
   };
+
+  static trimFile = params => {
+    console.log('trimFile FILE_TRIMMER_API', FILE_TRIMMER_API);
+    return fetch(`${FILE_TRIMMER_API || 'https://trim.kab.sh/rest/trim'}?${Requests.makeParams(params)}`)
```
Comment (davgur) [inline src/helpers/Api.js:257]: hardcoded url

**Comment by bbfsdev** [comment]
image

- Clicking in this state will close the pop-up and I will loose any option to get to the file link.
- Very not comfortable.
- This widget should work in background and show somewhere on top such that it will not disturb using archive and person can do anything else.

**Comment by bbfsdev** [comment]
image

- While sharing in hebrew we should keep the left to right! Start from left, End from right as similar to player direction left to right.
- PLEASE FIX THE 9:9 and 2:4 to be 2:04 and 9:09 it is so confusing.... note that it should be  1:09:09 if more then one hour.

**Review by bbfsdev — APPROVED** [review]

File: src/components/Pages/Unit/widgets/Downloads/MediaDownloads.js [hunk 3]
```diff
@@ -211,15 +211,12 @@ class MediaDownloads extends Component {
...
             compact
-            fluid
             as="a"
             href={url}
             target="_blank"
-            className="media-downloads__file-download-btn"
             size="mini"
             color="orange"
             content={ext.toUpperCase()}
-            onClick={() => chroniclesAppend('download', { url, uid: file.id })}
```
[the remark this answers is not in the document]
Comment (davgur) [reply src/components/Pages/Unit/widgets/Downloads/MediaDownloads.js:222]: not clear how its happen, restored

File: src/components/AVPlayer/Share/CutAndDownload.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      dir={dir}
+      trigger={
+        <Popup
+          open={openIconHover && !open}
+          onOpen={() => setOpenIconHover(true)}
+          onClose={() => setOpenIconHover(false)}
+          content={t('player.download.iconHoverText')}
+          position={'top right'}
+          trigger={
+            <Button
```
[the remark this answers is not in the document]
Comment (davgur) [reply src/components/AVPlayer/Share/CutAndDownload.js]: added

**Pull request merged**